test/test.combokeys.js

```diff
@@ -186,20 +186,28 @@ describe('combokeys.bind', function () {
 
       KeyEvent.simulate('S'.charCodeAt(0), 83, ['meta'], element)
 
-      expect(spy.callCount).to.equal(1, 'callback should fire')
-      expect(spy.args[0][0]).to.be.an.instanceOf(Event, 'first argument should be Event')
-      expect(spy.args[0][0].cancelBubble).to.be.True
```

Comment:
For some reason (probably a bug in chai), while `cancelBubble` was `false`, this passed, as if it was `true`.